### PR TITLE
Added setBrakeMode for chassis

### PR DESCRIPTION
**Pull request by dhruviyengar**
#### Summary
In EZ-Template, there's a feature to switch all the drive motor's brake modes. This is used for switching from hold for autonomous to coast in user control.

#### Motivation
This saves 1 line.

#### Test Plan
Tested on robot, switch occurred perfectly. RC1

#### Additional Notes
N/A

## Download the template for this pull request: 

> [!NOTE]  
> This is auto generated from [`Add Template to Pull Request`](https://github.com/LemLib/LemLib/actions/runs/7205705021)
- via manual download: [LemLib@0.5.0-rc3+121bac.zip](https://nightly.link/LemLib/LemLib/actions/artifacts/1113965097.zip)
- via PROS Integrated Terminal: 
 ```
curl -o LemLib@0.5.0-rc3+121bac.zip https://nightly.link/LemLib/LemLib/actions/artifacts/1113965097.zip;
pros c fetch LemLib@0.5.0-rc3+121bac.zip;
pros c apply LemLib@0.5.0-rc3+121bac;
rm LemLib@0.5.0-rc3+121bac.zip;
```

### CODE REVIEW

**Comment by Rocky14683** [comment]
that function shouldn't be taking mutex

**Comment by dhruviyengar** [comment]
> that function shouldn't be taking mutex

Fixed in https://github.com/LemLib/LemLib/pull/119/commits/6e748f119d4869836aadcb22b3ab4914c6a4b6da

File: include/lemlib/chassis/chassis.hpp [hunk 1]
```diff
@@ -206,6 +206,14 @@ class Chassis {
          * @param maxSpeed the maximum speed the robot can turn at. Default is 127
          * @param async whether the function should be run asynchronously. true by default
          */
+
+        void setBrakeMode(pros::motor_brake_mode_e mode);
+        /**
+         * @brief Sets the brake mode of the drivetrain motors
+         *
+         * @param mode Mode to set the drivetrain motors to
+         */
```
Comment (liam-teale) [inline include/lemlib/chassis/chassis.hpp]:
```suggestion
         /**
         * @brief Sets the brake mode of the drivetrain motors
         *
         * @param mode Mode to set the drivetrain motors to
         */
        void setBrakeMode(pros::motor_brake_mode_e mode);
```

**Review by liam-teale — CHANGES_REQUESTED** [review]
Just fix the location of this comment

File: include/lemlib/chassis/chassis.hpp [hunk 2]
```diff
@@ -260,6 +260,14 @@ class Chassis {
          * @param maxSpeed the maximum speed the robot can turn at. Default is 127
          * @param async whether the function should be run asynchronously. true by default
          */
+
+         /**
```
Comment (liam-teale) [inline include/lemlib/chassis/chassis.hpp]:
```suggestion
        /**
```

**Review by liam-teale — CHANGES_REQUESTED** [review]

**Pull request merged**